admin/orders.php

```diff
@@ -648,7 +648,9 @@ function couponpopupWindow(url) {
             <table class="table" id="addressDelivery">
               <tr>
                 <td><strong><?php echo ENTRY_SHIPPING_ADDRESS; ?></strong><br>
+                    <?php if (!empty($order->delivery)) { ?>
                     <button type="button" class="btn btn-xs btn-default mt-3" title="<?= TEXT_COPY ?>" onclick="copyToClipboard('delivery', this)"><?= TEXT_COPY ?></button>
+                    <?php } ?>
                 </td>
                 <td><?php echo (empty($order->delivery)) ? TEXT_NONE : zen_address_format($order->delivery['format_id'], $order->delivery, 1, '', '<br>'); ?></td>
```

Comment:
closing bracket of empty check should be at the end

Reply:
You mean the `(empty($order->delivery))` , right?

No, since the PHP statement is a simple ternary with a simple if-this-else-that logic that's not concatenating multiple strings or functions in the "this" or "else" parts, the true/false check can stand on its own before the opening `?`. Simpler to  read and maintain that way, because you don't have to worry about a dangling parenthesis at the end.

Technically it could be reduced to `empty($order->delivery)` without the wrapping parentheses because there's only one condition checked, but the wrapping parentheses does make it clear to the reader what the entire condition is.